.codegen/api.go.tmpl

```diff
@@ -12,7 +12,92 @@ import (
 	"github.com/databricks/databricks-sdk-go/listing"
 	"github.com/databricks/databricks-sdk-go/useragent"
 )
+{{ $extendedInterface :=
+	list
+		"Clusters"
+		"CommandExecution"
+		"Dbfs"
+		"Libraries"
+		"StatementExecution"
+		"Workspace"
+}}
 {{range .Services}}
+
+type {{.PascalName}}APIInterface interface {
```

Comment:
IMO we should call this just `Interface`, or even drop the suffix and use just `{{.PascalName}}` for brevity.

Reply:
+1

Reply:
Just using {{.PascalName}} unfortunately conflicts with some other names defined, for example `WorkspaceConf`. Will keep it with `Interface` name

Reply:
Thanks for checking!